packages/core/package.json

```diff
@@ -8,6 +8,7 @@
   "main": "main/index.js",
   "module": "module/index.js",
   "typings": "types/index.d.ts",
+  "type": "module",
```

Comment:
I've had issues with this `type:module`, can you explain why you want to do this?

this will make it incompatible with other modules.

Reply:
yea `type:module` should be removed. I've removed it